cmd/list-app-role-assignments.go

```diff
@@ -96,13 +98,15 @@ func listAppRoleAssignments(ctx context.Context, client client.AzureClient, serv
 					} else {
 						log.V(2).Info("found app role assignment", "roleAssignments", item)
 						count++
-						out <- AzureWrapper{
+						if ok := pipeline.Send(ctx.Done(), out, interface{}(AzureWrapper{
```

Comment:
Why did this one need an explicit conversion to an empty interface? If it's absolutely necessary, we should at least follow our newer conventions and use `any()` for the conversion, but I'm not seeing how it is necessary here.

Reply:
Looking at some related files, it looks like we typically use `NewAzureWrapper()` rather than directly creating an `AzureWrapper` struct, as that struct is deprecated. Perhaps now is the time to convert this to the newer type?